recipe/conda_build_config.yaml

```diff
@@ -21,5 +21,5 @@ pin_run_as_build:
   boost-cpp:
     max_pin: x.x.x
 docker_image:                                   # [linux64]
-  - quay.io/condaforge/linux-anvil-cos7-x86_64  # [linux64]
+  - quay.io/condaforge/linux-anvil-comp7  # [linux64]
```

Comment:
I wonder what this change really does - what's the difference between the two images?

Reply:
According to
https://quay.io/repository/condaforge/linux-anvil-comp7/manifest/sha256:93c7a11c8580cc709a154fb7bb4723dea9f466bfc0859df533560317a6d61794
https://quay.io/repository/condaforge/linux-anvil-cos7-x86_64/manifest/sha256:aec245b91a298c9ed32b73d1438ad79e7f71906a94e71b0efef2ff691685a3f2
linux-anvil-comp7 contains Centos 6 and linux-anvil-cos7-x86_64 contains Centos 7.